### PR TITLE
#74644 Use Updated jQuery Timepicker to Avoid Conflicts

### PR DESCRIPTION
**Pull request by ggwicz**
**Ticket:** [**#74644**](http://central.tri.be/issues/74644)

**Related PR:** **https://github.com/moderntribe/tribe-common/pull/535**

### CODE REVIEW

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -1953,10 +1953,6 @@ public function add_admin_assets() {
 				// date picker
 				Tribe__Events__Template_Factory::asset_package( 'datepicker' );
 
-				// jQuery Timepicker
-				wp_enqueue_script( 'tribe-jquery-timepicker' );
-				wp_enqueue_style( 'tribe-jquery-timepicker-css' );
```
Comment (borkweb) [inline src/Tribe/Main.php]: Shouldn't this still be enqueued somewhere?

File: src/resources/js/events-admin.js [hunk 2]
```diff
@@ -91,7 +91,11 @@ var tribe_timepickers = tribe_timepickers || {};
 				opts.forceRoundTime = true;
 			}
 
-			$item.timepicker( opts ).trigger( 'change' );
+			if ( 'undefined' !== typeof $.fn.tribeTimepicker ) {
+				$item.tribeTimepicker( opts ).trigger( 'change' );
+			} else {
+				$item.timepicker( opts ).trigger( 'change' );
```
Comment (borkweb) [inline src/resources/js/events-admin.js]: Add a comment to remove this failover in the future?

```php
// @deprecated TBD
```

File: src/Tribe/Main.php (hunk 1 above)
Comment (ggwicz) [reply src/Tribe/Main.php]: @borkweb yes, but it gets loaded in the "Asset package" for `events-admin.js` when it's called via `tribe_asset()`

See here: https://github.com/moderntribe/the-events-calendar/blob/7084c49219f875252cb5a219f07c88444d5445e6/src/Tribe/Asset/Admin.php#L17

**Comment by ggwicz** [comment]
this one's ready for another review!

File: src/resources/js/events-admin.js [hunk 3]
```diff
@@ -25,11 +25,11 @@ var tribe_timepickers = tribe_timepickers || {};
...
 	obj.container = function( k, container ) {
-		var $container = $( container ),
-			$all_day = $container.find( obj.selector.all_day ),
+		var $container  = $( container ),
+			$all_day    = $container.find( obj.selector.all_day ),
 			$timepicker = $container.find( obj.selector.timepicker ),
-			$timezone = $container.find( obj.selector.timezone ).not( obj.selector.input ),
-			$input = $container.find( obj.selector.timezone ).filter( obj.selector.input ),
+			$timezone   = $container.find( obj.selector.timezone ).not( obj.selector.input ),
+			$input      = $container.find( obj.selector.timezone ).filter( obj.selector.input ),
```
Comment (bordoni) [inline src/resources/js/events-admin.js]: can we split these into multiple `var` statements?

File: src/resources/js/events-admin.js [hunk 4]
```diff
@@ -70,7 +70,7 @@ var tribe_timepickers = tribe_timepickers || {};
 		// Setup all Timepickers
 		$timepickers.each( function() {
 			var $item = $( this ),
-				opts = $.extend( {}, obj.timepicker.opts );
+				opts  = $.extend( {}, obj.timepicker.opts );
```
Comment (bordoni) [inline src/resources/js/events-admin.js]: can we split these into multiple var statements?

**Pull request merged**